Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -995,6 +995,11 @@
     frame->loader().changeLocation(url, emptyAtom(), coreEvent.get(), ReferrerPolicy::NoReferrer, ShouldOpenExternalURLsPolicy::ShouldAllow);
 }
 
+RefPtr<PDFPluginAnnotation> PDFPluginBase::protectedActiveAnnotation() const
```

Comment:
Needs a PLATFORM(MAC) (I keep forgetting all the annotation code is #ifdeffed)

Reply:
Maybe a more specific one :)?

Reply:
(ideally, yes, but this is what the rest of the code does)